backend/cypress/e2e/audit-findings.cy.js

```diff
@@ -1,35 +1,39 @@
 import 'cypress-file-upload';
 
 describe('Audit findings page', () => {
-    before(() => {
-      cy.visit('/report_submission/audit-findings/2022XB40001000002');
-    });
-    it('Page loads successfully', () => {
-      cy.url().should('include','/report_submission/audit-findings/2022XB40001000002');
-    });
+  const reportTestId = '2023MAY0001000001'
```

Comment:
**question**: I don't understand how the actual audit with this ID gets created in the first place. Moving forward with completely automated tests, do we need to use some kind of Cypress command to create the initial audit and get the ID of it?

Reply:
Initially, the report id is added after a one completes a create new audit then I manually enter that ID into Cypress. I prefer not to do it this way, I would ideally like to create a Cypress Command to bypass that, so yes.